sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/debug/DebugConfig.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.debug;
+
+/**
+ * Determines if the SDK is in debugging mode (captures stack traces) or not.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public final class DebugConfig {
+  private static final String ENABLE_METRICS_DEBUG_PROPERTY = "io.opentelemetry.sdk.metrics.debug";
```

Comment:
Shouldn't be follow the spec's recommendation for [language specific environment variables](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md)?

I.e. something like `otel.java.sdk.metrics.debug`, or `otel.experimental.sdk.metrics.debug`.

Reply:
Yes.  For now I duplicated what was used for `Context` API (which predated the specification).  Will update.